### DISCUSSION TITLE
IBAX Airdrop 2. Kısmı Başlattı. Peki?

### DISCUSSION THREAD

**Discussion by Modelactor** [IBAX-io/go-ibax#1924] [Soru Cevap - Türkçe]
IBAX Airdrop 2. Raundu Başlattı.
Peki İlk kısımda mağdur olan IBAX takipçileri için bir telafi düzenlenecek mi?

**Comment by yolandadadada** [comment]
Could you speak English?

**Comment by ysfcndgr** [comment]
Kim mağdur oldu ?

**Reply by enginkose** [reply]
Ben

**Reply by frubtion** [reply]
Airdrop'un son günü 5000 puanı vermedi

**Reply by enginkose** [reply]
AirDrop rarihlerinde görevleri yaptım reputation yazan yerde 8,774 yazıyor bu nedir acab

**Reply by ysfcndgr** [reply]
> 

5000 Puan kimseye verilmedi, duyuruları takip etmenizi öneririm twitter APIsinden kaynaklanan sıkıntıdan dolayı o görevler gerçekleşmedi ve bunu yapanlarında puanları geri alındı.

**Comment by mustafa0505** [comment]
I guess It will be fixed.

**Comment by The0ffle** [comment]
Evet sıkıntılı geçti

**Comment by alfa5541** [comment]
aynen bende de oldu alamadık puanları retweet yaptığım halde gelmedi banada
